src/ui/popup/miscellaneous-settings/index.tsx

```diff
@@ -0,0 +1,16 @@
+import {m} from 'malevic';
+import {isFirefox} from '../../../utils/platform';
+import {ViewProps} from '../types';
+import EnabledByDefaultGroup from './enabled-by-default';
+import InvertPDF from './invert-pdf';
+import SyncSettings from './Sync-settings';
+
+export default function MiscellaneousSettingsPage(props: ViewProps) {
+    return (
+        <section class="m-section">
+            <EnabledByDefaultGroup {...props} />
```

Comment:
This "Enabled by default" is one of the most important settings, I think it should be at the top of others. Maybe the "Site list" page will be on top and this option will be moved there.